mlir/include/mlir/Interfaces/CallInterfaces.td

```diff
@@ -18,10 +18,16 @@
 include "mlir/IR/OpBase.td"
 
 
-/// Interface for operations with arguments attributes (both call-like
-/// and callable operations).
-def ArgumentAttributesMethods {
-  list<InterfaceMethod> methods = [
+/// Interface for operations with result and argument attributes.
+def ArgAndResultAttrsOpInterface : OpInterface<"ArgAndResultAttrsOpInterface"> {
+  let description = [{
+    An operation that has argument and result attributes. This interface
+    provides functions to access and modify the argument and result
+    attributes of the operation.
+  }];
+  let cppNamespace = "::mlir";
+
+  let methods = [
```

Comment:
This is the place where I did the change for the reland. Before it was:

`list<InterfaceMethod> methods = [`